pandas/tests/groupby/test_function.py

```diff
@@ -1358,6 +1358,30 @@ def test_apply_to_nullable_integer_returns_float(values, function):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("min_count", [0, 10])
+def test_groupby_sum_mincount_boolean(min_count):
+    b = True
+    a = False
+    c = b
+    na = np.nan
+    dfg = pd.array([b, b, na, na, a, a, b, c], dtype="boolean")
+
+    df = DataFrame({"A": [1, 1, 2, 2, 3, 3, 1], "B": dfg[:-1]})
```

Comment:
nit: I see this was copied from a more generic test, should we clean up here? I.e. remove occurrences of `c` and `dfg[:-1] -> dfg`

Reply:
good catch, updated